### PR TITLE
Issue 1210a

### PR DESCRIPTION
**Pull request by prb112**

### CODE REVIEW

**Comment by JohnTimm** [comment]
Minor nit. My preference would be to name the methods ‘getProfiles()’ and ‘getProfileResources()’

**Comment by prb112** [comment]
> Minor nit. My preference would be to name the methods ‘getProfiles()’ and ‘getProfileResources()’

Easy fix, I can change.  Still testing locally

File: fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java [hunk 1]
```diff
@@ -270,4 +274,31 @@ private Resource getResource(FHIRRegistryResource registryResource, String url,
...
+                .forEach(r -> processResource(r, resourceTypeWithCanonicalUrls));
+        return resourceTypeWithCanonicalUrls;
+    }
+
+    private void processResource(FHIRRegistryResource registryResource, Map<String,Set<Canonical>> resourceTypeWithCanonicalUrls) {
+        String type = registryResource.getType();
+        resourceTypeWithCanonicalUrls.compute(type, (k,v) -> checkOrCreateSet(k,v,registryResource));
+    }
+
+    private Set<Canonical> checkOrCreateSet(String k, Set<Canonical> v,FHIRRegistryResource registryResource){
```
Comment (lmsurpre) [inline fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java:296]:
```suggestion
    private Set<Canonical> checkOrCreateSet(String k, Set<Canonical> v, FHIRRegistryResource registryResource) {
```

**Comment by JohnTimm** [comment]
> > Minor nit. My preference would be to name the methods ‘getProfiles()’ and ‘getProfileResources()’
> 
> Easy fix, I can change. Still testing locally

I meant getProfiles() in FHIRRegistry and getProfileResources() in FHIRRegistryResoirceProvider to match the existing getProfiles(String type) and getProfileResources(String type) respectively.

File: fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java (hunk 1 above)
Comment (prb112) [reply fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java:296]: Doing this locally..

Comment (prb112) [reply fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java:296]: Fixed

**Comment by prb112** [comment]
Changed per John's suggestion

**Pull request merged**